examples-standalone/dashboard/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + Vue</title>
```

Comment:
I was wandering if we could add here our icon and something that mentions Kendo

Reply:
I've updated the Meta title and changed the icon with one with a green Kendoka